src/JWT/JWTBuilder/Models/JWTData.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections.Generic;
+namespace JWT.JWTBuilder.Models
+{
+
+    /// <summary>
+    /// Represents the Data that will store in a JWT.
+    /// </summary>
+    public class JWTData
+    {
+        /// <summary>
+        /// Create a new instance of JWTData and initalize Header and Payload.
+        /// </summary>
+        /// <returns>A Instance of <see cref"JWTData"/></returns>
+        public JWTData() : this(Header: new Dictionary<string, string>(), PayLoad: new Dictionary<string, object>())
+        {
+        }
+
+        /// <summary>
+        /// Create a new instance of JWTData
+        /// </summary>
+        /// <param name="Header">A Instance of a dictionary that contains the headers</param>
+        /// <param name="PayLoad">A instance of a dictionary that contans the payload</param>
+        public JWTData(Dictionary<string, string> Header, Dictionary<string, object> PayLoad)
```

Comment:
We already have class for this - JwtParts. You can reuse it once merge the projects

Reply:
Hi @abatishchev I don't know if I can reuse it. I don't think this to classes represent the same stuff.

JWParts are more a think to make the work with a encoded token easier. But JWTDate is more for the decoded Token work. So you can easy add an remove data from the token befor you encode the token. 
I'm not shure if I can merge this two worlds together.

Because to get a JWTData from a JWTParts you need for example a URLDecoder.